src/linear_algebra/char_poly/coeff.lean

```diff
@@ -0,0 +1,191 @@
+/-
+Copyright (c) 2020 Aaron Anderson, Jalex Stark. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Aaron Anderson, Jalex Stark.
+-/
+
+import linear_algebra.char_poly
+import linear_algebra.matrix
+import ring_theory.polynomial.basic
+import algebra.polynomial.big_operators
+
+/-!
+# Characteristic polynomials
+
+We give methods for computing coefficients of the characteristic polynomial.
+
+## Main definitions
+
+- `degree_char_poly_eq_dim`
+- `det_from_char_poly`
+- `trace_from_char_poly`
+
+## Implementation details
+
+-/
+
+noncomputable theory
+
+universes u v w z
+
+open polynomial matrix
+open_locale big_operators
+
+variables {R : Type u} [comm_ring R]
+variables {n : Type v} [fintype n] [decidable_eq n]
+variables {α : Type v} [decidable_eq α]
+
+open finset
+open polynomial
+
+section fixed_points
+-- TODO: move this section to be with other combinatorial facts about the symmetric group
```

Comment:
It would be good to do this.

Reply:
(I agree but don't know where to put it.)

Reply:
We could shoehorn it into group_theory/perm/cycle.lean or make a new file in group_theory/perm

Reply:
We put these into group_theory/perm/cycle.lean, which seems vaguely suboptimal but also a lot better than char_poly/coeff.lean. Some good signs: I didn't have to add any imports to the target file, and the target file already opened namespaces that we were accessing with dot notation.